docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker support
+
+The files in this directory allow you to build a docker image.  The query coordinator assembly must be 
+copied to `query-coordinator-assembly.jar` in this directory before building.
+
+## Required Runtime Variables
+
+* `ZOOKEEPER_ENSEMBLE` - The zookeeper cluster to talk to, in the form of `["10.0.0.1:2181", "10.0.0.2:2818"]`
+* `SECONDARY_INSTANCES` - The list of all the secondary instances that this query coordinator should talk to.
+
+## Optional Runtime Variables
+
+See the Dockerfile for defaults.
+
+* `ARK_HOST` - The IP address of the host of the docker container, used for service advertisements.
```

Comment:
ARK_HOST is set by the socrata base image automatically.

On Mon, Feb 23, 2015 at 5:40 PM, John Krah <notifications@github.com> wrote:

> In docker/README.md
> <https://github.com/socrata/query-coordinator/pull/28#discussion_r25221735>
> :
>
> > @@ -0,0 +1,20 @@
> > +# Docker support
> > +
> > +The files in this directory allow you to build a docker image.  The query coordinator assembly must be
> > +copied to `query-coordinator-assembly.jar` in this directory before building.
> > +
> > +## Required Runtime Variables
> > +
> > +* `ZOOKEEPER_ENSEMBLE` - The zookeeper cluster to talk to, in the form of `["10.0.0.1:2181", "10.0.0.2:2818"]`
> > +* `SECONDARY_INSTANCES` - The list of all the secondary instances that this query coordinator should talk to.
> > +
> > +## Optional Runtime Variables
> > +
> > +See the Dockerfile for defaults.
> > +
> > +* `ARK_HOST` - The IP address of the host of the docker container, used for service advertisements.
>
> Dockerfile needs a default for ARK_HOST now
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata/query-coordinator/pull/28/files#r25221735>.
>